src/glmfit.jl

```diff
@@ -230,6 +247,41 @@ fit{M<:AbstractGLM}(::Type{M}, X::@compat(Union{Matrix,SparseMatrixCSC}), y::Abs
                     d::UnivariateDistribution, l::Link=canonicallink(d); kwargs...) =
     fit(M, float(X), float(y), d, l; kwargs...)
 
+"""
+    glm(X, y, args...; kwargs...)
+
+Fits the generalised linear model, specified by the symbolic description of the
```

Comment:
American English "generalized" is used everywhere else.

X isn't a symbolic description here AFAIK, it's the model matrix/design matrix.